android/app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java

```diff
@@ -161,6 +161,29 @@ public Session getSessionById(int id, SQLiteDatabase mDb) {
         return location;
     }
 
+    public List<Integer> getSpeakerIdbySessionId(int id, SQLiteDatabase mDb) {
+        String selection = DbContract.Sessionsspeakers.SESSION_ID + EQUAL + id;
+        Cursor cur = mDb.query(
+                DbContract.Sessionsspeakers.TABLE_NAME,
+                DbContract.Sessionsspeakers.FULL_PROJECTION,
+                selection,
+                null,
+                null,
+                null,
+                null
+        );
```

Comment:
@the-dagger Since you're not calling this method anywhere, we can remove this as well.